frontend/test/metabase/scenarios/filters/sql-field-filter-date.cy.spec.js

```diff
@@ -55,10 +57,12 @@ describe("scenarios > filters > sql filters > field filter > Date", () => {
     mockSessionProperty("field-filter-operators-enabled?", true);
```

Comment:
(Just an observation). I opened this file in my editor, and definitely felt like the big DATE_FILTER_SUBTYPES object could live in another file.

Reply:
Yup. Thanks.
That's the topic of https://github.com/metabase/metabase/pull/16893.